features/upgrade/node/scc.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Seccomp part of SCC policy should be kept and working after upgrade	
+
+  # @author sunilc@redhat.com
+  @upgrade-prepare
+  @admin
+  Scenario: Upgrade - Seccomp part of SCC policy should be kept and working after upgrade - prepare
+    Given I switch to cluster admin pseudo user
+    When I run the :new_project client command with:
+      | project_name | node-upgrade-scc |
+    And I use the "node-upgrade-scc" project
+    When I run the :create client command with:
```

Comment:
@sunilcio please use the step `Given I obtain...` to retrieve the testdata.  Please refer #1227 as example of how to do it.

Reply:
```
Given I obtain test data file "testdata/node/scc.yaml"
When I run the :create client command with:
  | f | testdata/node/scc.yaml |
```

Reply:
By the way, creating scc should be destructive, as scc can significantly affect a lot of scenarios.